api/endpoints/testdata/fixtures/responses/list_users.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Small nitpick, feel free to ignore: the formatting is a little bit inconsistent throughout the file, maybe use a JSON formatter to autoformat it.